website/content/en/docs/deploy-guide.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Development Guide"
+linkTitle: "Development Guide"
+weight: 50
+---
+
+# Resources Karpenter Expects/Creates
+
+- Cluster on AWS
```

Comment:
We need to be really clear that your cluster does not need to be on AWS.

Reply:
i thought we didn't support creating nodes in different region? but we support instances in a different cloud altogether?

Reply:
You can run/install Karpenter anywhere. It currently only launches EC2 instances.

Reply:
> We need to be really clear that your cluster does not need to be on AWS.

> [Karpenter] currently only launches EC2 instances.

I'm confused by this. How can karpenter only launch EC2 instances but yet support not running on AWS?